kuma/scrape/management/commands/__init__.py

```diff
@@ -18,12 +18,12 @@ def parse_url_or_path(self, url_or_path):
         if url_or_path.startswith('http'):
             bits = urlparse(url_or_path)
             host = bits.netloc
-            path = bits.path
+            path = bits.path[:-1] if bits.path.endswith('/') else bits.path
```

Comment:
Thanks for this @joedarc! What you have of course works fine, but I was thinking a simpler approach might be to use `path = bits.path.rstrip('/')`.

Reply:
Agreed, that is cleaner. Pushed up the change 👍 .